Examples/Framework/include/ActsExamples/EventData/SimSpacePoint.hpp

```diff
@@ -58,15 +60,15 @@ class SimSpacePoint {
   // Variance in rho/z of the global coordinates
   float m_varianceRho;
   float m_varianceZ;
-  // Index of the corresponding measurement
-  Index m_measurementIndex;
+  // Indices of the corresponding measurements
+  std::vector<Index> m_measurementIndices;
```

Comment:
What about using `boost::container::static_vector<Index, 2>` instead of `std::vector<Index>`?